docs/rtd/getting_started.rst

```diff
@@ -22,18 +22,16 @@ for example need to provide correct types of parameters to module function calls
 Modules
 =======
 
-YARA is easily extensible with modules which provide you way to call C functions from your YARA ruleset. We realize that modules are important
-and yaramod therefore supports them. You might however run into functions which you do not recognize and can' find anywhere in upstream YARA.
-We at Avast use YARA daily and sometimes there are things we would like to match in our YARA rules something what is not accessible to the outside
-world so we improve existing or develop our own modules. What we can we share with the community and in the future, we would really like to see
-all the functions we have also in upstream YARA but that doesn't entirely depend on us. It's not like we are keeping it away becuase we want to keep
-it secret.
+YARA is easily extensible with modules which provide you a way to call C functions from your YARA ruleset. We realize that modules are important
```

Comment:
Can you also add a section summarizing the different locations that yaramod can consume? I have looked at the code and it seems you can specify the location using `YARAMOD_MODULE_SPEC_PATH` env variable, but I wasn't able to find it here.

What I suggest is creating some kind of short list or hierarchy, that will go through the priority, for example:
1. `YARAMOD_MODULE_SPEC_PATH` env variable
1. Directory passed to yaramod instance constructor - will load custom `.json` files
1. Upstream yara default - builtin to yaramod

Also please state the overwriting policy, as to what happends when you use custom modules. Are other things on the hierarchy ignored?

Reply:
Hello and thank you for all your remarks. I added a section that should clear out both the preference and overwriting.